### PR TITLE
improve choice of vurb

### PR DESCRIPTION
**Pull request by MirjamvT**

### CODE REVIEW

**Comment by bvhme** [comment]
Hey @MirjamvT, could you explain a bit why this would be better? The vague 'approach' seems to me be quite confusing while 'fulfil' is clear about this being a goal for the requirements.

**Comment by ElenaFdR** [comment]
@bvhme @MirjamvT had a chat with me about this. She's concerned that 'fulfill' sounds like the information on the page will be complete and exhaustive, whereas in practice it's neither necessary nor sufficient to 'fulfill the requirements of your employment'.

**Comment by bvhme** [comment]
I understand. It should more of give a starting point. Perhaps we can
rephrase it to be in the area of 'explains: how to do your work and get
started ...' or something like that...

**Comment by bvhme** [comment]
Whenever you're back from Holiday @MirjamvT, let's look at this again.

File: contributor-guides/for-staff.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 type: Guide
-explains: How to fulfill the requirements of your employment with the Foundation for Public Code
+explains: How to approach the requirements of your employment with the Foundation for Public Code
```
Comment (bvhme) [inline contributor-guides/for-staff.md]:
```suggestion
explains: The basics for being a staff member of the Foundation for Public Code
```
